rerun_notebook/callbacks.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ec34bdbe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c7dedfe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%env RERUN_NOTEBOOK_ASSET=inline"
```

Comment:
Huh, this makes it nicer for development, I forgot that var and it just set it for me. Question: We can't really set this in an "offical" example like `example.ipynb` because that would mess with using that example with a public Rerun install, right? I.e. when we polish up this sample for public consumption this has to go?

Reply:
https://github.com/rerun-io/rerun/issues/9274